<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creditwindow</class>
 <widget class="QDialog" name="Creditwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 224, 255);
border-radius: 20px;</string>
  </property>
  <widget class="QLineEdit" name="credit_sum_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>25</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="credit_sum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="credit_term">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Срок кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit_term_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="credit_rate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Кредитная ставка, % годовых</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit_rate_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="repayment_procedure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Порядок погашения</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_payments">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Всего платежей:</string>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Переплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_payment_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   0</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_payments_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   0</string>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>451</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 84, 179);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_payment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Маскимальный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="min_payment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   Минимальный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="min_payment_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 164, 245);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>   0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>221</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="year">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
      </property>
      <property name="text">
       <string>года</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="mounth">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
      </property>
      <property name="text">
       <string>месяца</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>221</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="annuity">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
      </property>
      <property name="text">
       <string>Аннуитетный</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="differentiated">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 127, 209);
border-radius: 7px;</string>
      </property>
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
